website/docs/docs/configure-dbt-extension.md

```diff
@@ -0,0 +1,207 @@
+---
+title: "Configure the dbt VS Code extension"
+sidebar_label: "Configure dbt extension"
+id: "configure-dbt-extension"
+description: "Prepare your local environment (files, env vars, connectivity) so the VS Code extension and terminal both work smoothly."
+---
+
+Whether you currently use <Constant name="dbt_platform" /> or self-hosted <Constant name="core" />, use this page to configure local setup for the dbt VS Code extension and configure environment variables, if your project uses them. 
+
+If you're new to dbt or getting started with a new project, you can skip this page and check out our [Quickstart for the <Constant name="fusion_engine"/>](/guides/fusion?step=1) to get started with the dbt extension.
+
+The steps differ slightly depending on whether you use <Constant name="dbt_platform" /> or <Constant name="core" />.
+
+- <Constant name="dbt_platform" /> &mdash; You’ll mirror your <Constant name="dbt_platform" /> environment locally to unlock <Constant name="fusion" />-powered features like <Constant name="mesh" />, deferral, and so on. You'll also set up environment variables to leverage the VS Code extension's features.
+- <Constant name="core" /> &mdash; You’ll confirm that your existing local setup and environment variables work seamlessly with the <Constant name="fusion_engine" /> and VS Code extension.
+
+## Prepare your local environment
+
+This section explains how to prepare your local environment to mirror your dbt project environment. If you already have your local environment set up, you don't need to do anything here. 
```

Comment:
It feels like we're using "environment" in two different ways here (local dbt installation vs. cloud staging/prod/development environments), which may be a little confusing. Maybe we could substitute "local environment" with "self-hosted dbt" or something similar
```suggestion
The following instructions show how to configure your self-hosted dbt project to mirror your <Constant name="dbt_platform" /> project environment: 
```